modules/registry-operator-configuration-resource-overview-openstack-swift.adoc

```diff
@@ -13,27 +13,27 @@ registry storage.
 |Parameter |Description
 
 |`authURL`
-|This value is optional.
+|Defines the URL for obtaining the authentication token. This value is optional.
 
 |`authVersion`
-|This value is optional.
+|Specifies the Auth version of {rh-openstack}. This value is optional.
```

Comment:
Does a typical user know what the Auth version means or what the value is?

Reply:
I believe that user coming to this table knows the Auth version but will check to be sure. 
@xiuwang Do we need to give an example of Auth version here or explain what Auth version is?

Reply:
@xenolinux @mburke5678 Sorry for the delayed response, I missed this message. 
Here is an example for the authVersion
```
        authURL: https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3
        authVersion: "3"
```
Please let me know if you still have concern.

Reply:
Thank you. I added the `authversion` example in the description.